tests/test_messaging.py

```diff
@@ -728,7 +728,7 @@ def test_has_equal_value_dont_wrap_newline():
     assert output["message"] == "Check your call of <code>print()</code>. Did you correctly specify the first argument? Expected something different."  # nopep8
 
 
-def test_has_equal_value_dont_wrap_too_long():
+def test_has_equal_value_shorten_too_long():
```

Comment:
A test for when the solution is long but the student code short would also be use-full.